content/governing-board/elections/2024/nominees.toml

```
@@ -332,10 +332,11 @@ Many things need to come together to achieve this, and we certainly don't have a
 We hope to bring the viewpoint of a for-profit company into the GB and help make the Matrix attractive for commercial businesses for the mutual benefit of the companies, and the community."""
 
 [[silver_members]]
-name = "[placeholder]"
+name = "Jan Krammer"
+pronouns = "he/him"
 project = "fairkom society"
-bio = "This is a placeholder which will be replaced with the actual nominee's information, pending an outstanding question."
-
+bio = "After my first FOSS experience with Firefox and Linux at an early age, there was no going back. When I was young and needed the money (while studying physics), I started working for our university and got more into servers, hosting and deploying FOSS stacks like jitsi and matrix. I then became independent while still working closely together with fairkom and universities focusing on opensource. I am a Sci-fi nerd who loves jazz piano and I am currently enjoying my thirties in Vienna."
```

```suggestion
bio = "After my first FOSS experience with Firefox and Linux at an early age, there was no going back. When I was young and needed the money (while studying physics), I started working for our university and got more into servers, hosting and deploying FOSS stacks like Jitsi and Matrix. I then became independent while still working closely together with fairkom and universities focusing on opensource. I am a Sci-fi nerd who loves jazz piano and I am currently enjoying my thirties in Vienna."
```

I think both Jitsi and Matrix commonly use the capital first letter in their Names :) (I checked fairkom too but they use lowercase for all of it :) )